.gitignore

```diff
@@ -138,3 +138,5 @@ nohup.out
 #sqlite3
 db.sqlite3
 
+# Pycharm
+.idea/
```

Comment:
This change and the removal of the `.idea/*` files should appear in a different PR. Please extract this out of this change.